api/v1/server/handlers/v1/workflow-runs/get.go

```diff
@@ -88,7 +93,13 @@ func (t *V1WorkflowRunsService) getWorkflowRunDetails(
 		return nil, err
 	}
 
-	result, err := transformers.ToWorkflowRunDetails(taskRunEvents, workflowRun, shape, tasks, stepIdToTaskExternalId, workflowVersion)
+	externalIdToPayload, err := t.config.V1.OLAP().ReadPayloads(ctx, tenantId, externalIds...)
```

Comment:
I don't love drilling this down into the transformer but I didn't want to modify a bunch of the structs we pass around to include payloads here since it'd be a much bigger change. I think we'll need to refactor this stuff anyways, so I figured the tech debt here was fine